File: Voca/Classes/Item.cs
﻿namespace Voca
{
	public class Item
	{
		public Item()
		{
		}

		public Item(string key, string value)
		{
			Key = key;
			Value = value;
		}

		public string Key { get; set; }

		public string Value { get; set; }
	}
}

File: Voca.Android/app/src/main/java/com/voca/testing/classes/Item.java
package com.voca.testing.classes;

public class Item {

    private String _source;
    private String _translate;

    public Item(String source, String translate) {
        _source = source;
        _translate = translate;
    }

    public String get_source() {
        return _source;
    }

    public String get_translate() {
        return _translate;
    }
}
